grails-app/controllers/org/bbop/apollo/OrganismController.groovy

```diff
@@ -44,11 +44,8 @@ class OrganismController {
             log.debug "deleteOrganism ${organismJson}"
             if (permissionService.isUserAdmin(permissionService.getCurrentUser(organismJson))) {
 
-                log.debug "organism ID: ${organismJson.id}"
-                Organism organism = Organism.findById(organismJson.id as Long)
-                if (!organism) {
-                    organism = Organism.findByCommonName(organismJson.organism)
-                }
+                log.debug "organism ID: ${organismJson.id} vs ${organismJson.organism}"
+                Organism organism = Organism.findById(organismJson.id as Long) ?: Organism.findByCommonName(organismJson.organism)
```

Comment:
I thought about using it here, but in one case it uses organismJson.id and in the other it uses organismJson.organism so I can't pass the same token in.    But I like this change.